Tasks/DownloadPackageV1/task.json

```diff
@@ -0,0 +1,188 @@
+{
+    "id": "e9e6165a-f8e3-4341-838f-cde7a2f23195",
+    "name": "DownloadPackage",
+    "friendlyName": "Download Package",
+    "description": "Download a package from a Package Management feed in Azure Artifacts or TFS. \r\n Requires the Package Management extension.",
+    "helpUrl": "",
+    "helpMarkDown": "Needs Package Management extension to be installed",
+    "category": "Utility",
+    "author": "ms-vscs-rm",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
```

Comment:
Do make the minor version 148 - it makes cherry picking task fixes a much less painful experience :)